DependencyInjection/Configuration.php

```diff
@@ -215,6 +215,8 @@ private function addFilesystemSection(ArrayNodeDefinition $node)
                                     ->end()
                                 ->end()
                                 ->scalarNode('region')->defaultValue('s3.amazonaws.com')->end()
+                                ->scalarNode('version')->defaultValue('latest')->end()
```

Comment:
Can you explain the usage of this settings ? Maybe you should add some note in the documentation.